blocks/link-list/link-list.js

```diff
@@ -5,40 +5,61 @@ import { isExternalURL } from '../../blocks-helpers/isExternalURL.js';
  * @function
  * @param {string} textContent the desired link text
  * @param {string} url the desired link href
+ * @param {string} [altText] the link title
```

Comment:
Alt text needs to use an `aria-label` attribute not a `title` attribute. the `title` attribute is inconsistent with screen readers, and sometimes read in addition to the link text. `aria-label` instead replaces the text content of the anchor element for a better experience.